src/librustc/ty/print/pretty.rs

```diff
@@ -213,6 +213,23 @@ pub trait PrettyPrinter<'tcx>:
         Ok(self)
     }
 
+    /// Prints `{...}` around what `f` and optionally `t` print
```

Comment:
I would put parens around "and optionally `t`" because I had a hard time reading it at first.